docs/DeveloperGuide.md

```diff
@@ -213,71 +236,6 @@ The following activity diagram summarizes what happens when a user executes the
 
 ![FilterActivityDiagram](images/FilterActivityDiagram.png)
 
-### \[Proposed\] Undo/redo feature
```

Comment:
Should we leave this undo/redo feature in as a 'proposed' idea or remove it completely ?